src/status_im2/contexts/chat/menus/pinned_messages/view.cljs

```diff
@@ -51,7 +52,7 @@
          :render-fn   message-render-fn
          :footer      [rn/view {:style (style/list-footer bottom-inset)}]
          :key-fn      list-key-fn
-         :separator   quo/separator}]
+         :separator   [quo/separator {:style {:margin-vertical (:tiny spacing/spacing)}}]}]
```

Comment:
I don't quite know what this `quo/separator` component is. I can't quite see it in the design system. Perhaps it should be just moved to status_im2.common?

Reply:
Yes, I'll move it to status_im2.common. That should be a good idea. Thank you!!